launch/world_modeling/people_recognition.launch

```diff
@@ -13,10 +13,8 @@
     <arg name="openface_srv_name" default="face_recognition/recognize" />
     <arg name="openpose_srv_name" default="/$(arg robot_name)/openpose/recognize" />
     <arg name="keras_srv_name" default="face_recognition/get_face_properties" />
-    <arg name="enable_topic_mode" default="False" />
-    <arg name="camera_info_depth" default="/$(arg robot_name)/head_rgbd_sensor/depth_registered/camera_info" />
-    <arg name="image_depth" default="/$(arg robot_name)/head_rgbd_sensor/depth_registered/image" />
-    <arg name="image_rgb" default="/$(arg robot_name)/head_rgbd_sensor/rgb/image_raw" />
+
```

Comment:
Why would you get rid off topic mode, this was very nice for demo's!

Reply:
Topic mode is still there, it is just in a parameter file. But perhaps that it is better to define this one in the launch file

Reply:
Configuration should be in one spot, so move it all to the config file IMO.

Reply:
Topic mode can change depending on the challenge/demo we run. The other parameters are for tuning. IMO we don't want to keep two near identical config files because one uses topic mode and the other does not.